code/restful-api/restful-api/Models/Participa.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace RestfulApi.Models
+{
+    public class Participa
```

Comment:
Maybe we should name this `UsuarioSorteio` if we decide to go with this naming approach.

Reply:
Feito.